src/actions/location-actions.js

```diff
@@ -158,6 +158,8 @@ export const getLocation = (locationId) => async (dispatch, getState) => {
 
   const params = {
     expand: "rooms,floors,images,maps",
+    fields:
+      "rooms.id,rooms.name,rooms.capacity,rooms.floor_id,floors.id,floors.name,floors.number,maps.id,maps.name,images.id,images.name",
```

Comment:
@santipalenque we need to get rid of the **published_events** collection at root level and at rooms
please check screen cap
<img width="885" alt="Screenshot 2024-10-24 at 16 23 37" src="https://github.com/user-attachments/assets/d300c72b-9e06-49ed-a811-78f36d716e33">